File: API/Entities/Exams.cs
using System.ComponentModel.DataAnnotations;
namespace API.Entities

{
    public class AppExams
    {
        [Key]
        public int ExamId { get; set; }

        public string ExamName { get; set; }

        public string ExamDescription { get; set; }

        public int ExamGrade { get; set; }

        public ExamPhoto ExamPicture { get; set;}

        public List<Question> ExamQuestions { get; set;}  

        public ICollection<UserExam> UserExams { get; set;}
    }
}

File: client/src/app/_models/exams.ts
import { ExamPicture } from "./exampicture";
import { ExamQuestions } from "./examquestions";
import { UserExam } from "./userexam";

export interface Exams {
    examId: number;
    examName: string;
    examDescription: string;
    examGrade: number;
    examPictureUrl: string;
    examPicture: ExamPicture;
    examQuestion: ExamQuestions[]
    userExams: UserExam[];
}

